merlin/models/tf/transforms/features.py

```diff
@@ -63,6 +63,8 @@ def compute_output_shape(self, input_shape):
                 col_schema_shape = self.schema[name].shape
                 if col_schema_shape.is_list:
                     max_seq_length = col_schema_shape.dims[1].max
+                    if max_seq_length is not None:
+                        max_seq_length = int(max_seq_length)
```

Comment:
When serializing/deserializing the schema, we get back a float value of the shape. If/when we fix that in core, this line can be removed